src/Value/Type/AbstractUuid.php

```diff
@@ -31,6 +31,16 @@ public static function fromBytes($bytes)
         return new static(Uuid::fromBytes($bytes));
     }
 
+    /**
+     * @param string $namespace Must be a valid uuid on itself
```

Comment:
What happens (is thrown) if it isn't?

Reply:
An exception is thrown indeed, will add it to the docblock